pcdsdevices/valve.py

```diff
@@ -858,3 +858,42 @@ class VCN_VAT590(BaseInterface, Device):
     )
 
     status = Cpt(VCN_VAT590_Status, '',)
+
+
+class VFV(Device):
+    """
+    VFV = Variable Frequency Valves
+
+    It corresponds to ST_VCN in the lcls-twincat-vacuum library.
```

Comment:
Is this true? It uses the same struct as the VCN class and we need a different pcdsdevices class for it? That seems wild to me.

Reply:
Jing suggested we renamed the class to VCN_OpenLoop. The device class will use the ST_VCN with open loop controls.